### DISCUSSION TITLE
Help with Perfect Prime TC0301 Thermometer

### DISCUSSION THREAD

**Discussion by tipp843** [artisan-roaster-scope/artisan#1349] [Q&A]
Hi,

I have just bought a Huky 500T and a Perfect Prime TC0301 Dual thermometer. I have loaded the latest software, download what I think are the drivers, and Artisan now recognises the meter but I still get the uu instead of temperature. To be fair, I'm not sure if I should be configuring a "device" or "machine" and I don't see Perfect Prime in either. I was recommended this model so I assume I'm just doing it wrong. The thermometer shows the correct temperatures so is working and when I connect it to the PC I can see and use the device software. I did make sure I closed down the device software before trying Artisan.

Any help would be gratefully received.

Many Thanks
Simon

**Comment by MAKOMO** [comment]
The Prime seems to be the same device sold under then name Amprobe TMPD-56 or Mastech MS6514. Just choose one of those under `meter` (menu Config >> Device, 1st tab)

**Reply by tipp843** [reply]
Will do and thanks...Just did a dodgy first roast without Artisan but will give it another go later and report back.

**Comment by tipp843** [comment]
Hi, I factory reset Artisan and then added device as Amprobe but still not working, is there somewhere that shows all the various settings for the different devices? Many Thanks Simon

**Reply by MAKOMO** [reply]
There are now device specific settings for devices on the Artisan side (but for Phidgets and Yocto modules). However, there is a section on _Omega HH806AU / Omega HH802U / Amprobe TMD-56_ in the [Artisan installation notes](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md) which might apply also to the Perfect Prime.

**Reply by tipp843** [reply]
Thanks, I did read that but it doesn't give the settings but just says to put them on channel zero. Guess I might have to send it back and just get one that's on the list!!! I didn't realise they were all different... Thanks
